packages/manager/src/features/VPC/VPCLanding/VPCLanding.tsx

```diff
@@ -121,7 +143,7 @@ const VPCLanding = () => {
         </TableHead>
         <TableBody>
           {vpcs?.data.map((vpc: VPC) => (
-            <VPCRow key={vpc.id} vpc={vpc} />
+            <VPCRow handleDeleteVPC={handleDeleteVPC} key={vpc.id} vpc={vpc} />
```

Comment:
Alternatively you could pass 
```suggestion
            <VPCRow handleDeleteVPC={() => handleDeleteVPC(vpc.id, vpc.label)} key={vpc.id} vpc={vpc} />
```

Which would clean up the Delete action in `VPCRow`